From a78a4607d96cdb49335672b927ff93e1f6e65351 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Tue, 2 Jan 2024 16:49:30 +0100
Subject: [PATCH 0305/1695] dts: stm32: add RIFSC compatible to enable it

RIFSC compatible was missing in its node. Add it so that there's a
match with the associated driver.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git e997afb029f1ced973997ddba4180a360599de33 dts: stm32: add RIFSC compatible to enable it]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I7c55d94c677d723999960dc09e3e4ea0e63c4265
---
 core/arch/arm/dts/stm32mp251.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/dts/stm32mp251.dtsi b/core/arch/arm/dts/stm32mp251.dtsi
index 729abaecc..cdea25840 100644
--- a/core/arch/arm/dts/stm32mp251.dtsi
+++ b/core/arch/arm/dts/stm32mp251.dtsi
@@ -118,6 +118,7 @@
 		};
 
 		rifsc: rifsc@42080000 {
+			compatible = "st,stm32mp25-rifsc";
 			reg = <0x42080000 0x1000>;
 			#address-cells = <1>;
 			#size-cells = <1>;
-- 
2.47.2

